Library/Homebrew/language/node.rb

```diff
@@ -14,16 +25,23 @@ def self.setup_npm_environment
       npmrc.write npm_cache_config
       # explicitly use our npm and node-gyp executables instead of the user
       # managed ones in HOMEBREW_PREFIX/lib/node_modules which might be broken
-      ENV.prepend_path "PATH", Formula["node"].opt_libexec/"npm/bin"
+      ENV.prepend_path "PATH", Formula["node"].opt_libexec/"bin"
     end
 
     def self.std_npm_install_args(libexec)
       setup_npm_environment
       # tell npm to not install .brew_home by adding it to the .npmignore file
       # (or creating a new one if no .npmignore file already exists)
       open(".npmignore", "a") { |f| f.write("\n.brew_home\n") }
+      pack = pack_for_installation
```

Comment:
Pedantry but I'd add a newline above this because the comment doesn't seem to reference it but yet it feels related.